tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -148,30 +155,59 @@ describe(__filename, () => {
     const addon = addonForGrouping(grouping);
     store.dispatch(setGroupedRatings({ addonId: addon.id, grouping }));
     const root = render({ addon });
-    const counts = root.find('.RatingsByStar-star').find(Link);
-    const bars = root.find('.RatingsByStar-barContainer').find(Link);
 
-    function validateLink(link, score, expectedTitle) {
-      expect(link).toHaveProp(
-        'to',
-        reviewListURL({ addonSlug: addon.slug, score }),
+    const tableRow = root.find('.RatingByStar-table-row');
+
+    function validateLink(row, score, expectedTitle) {
+      const clientApp = CLIENT_APP_ANDROID;
+      const lang = 'en-US';
+
+      expect(row).toHaveProp('onClick');
+
+      row.simulate('click', createFakeEvent({}));
```

Comment:
You don't need the `{}` here, it can just be `createFakeEvent()`.

Reply:
Yes,I will update this.